tests/output/ConfWriterTest.py

```diff
@@ -72,3 +80,31 @@ def test_write(self):
             lines = f.readlines()
 
         self.assertEqual(result_file, lines)
+
+    def test_write_with_dir(self):
+        result_file = ['[Default]\n',
+                       'save = true\n',
+                       'a_default, another = val\n',
+                       '# thats a comment\n',
+                       'test = push\n',
+                       't = \n',
+                       '[Section]\n',
+                       '[MakeFiles]\n',
+                       'j, ANother = a\n',
+                       'multiline\n',
+                       'value\n',
+                       '; just a omment\n',
+                       '; just a omment\n',
+                       'key\\ space = value space\n',
+                       'key\\=equal = value=equal\n',
+                       'key\\\\backslash = value\\\\backslash\n',
+                       'key\\,comma = value,comma\n',
+                       'key\\#hash = value\\#hash\n',
+                       'key\\.dot = value.dot\n']
+        self.uut_dir.write_sections(self.conf_parser.parse(self.file))
```

Comment:
instead of parsing that file (the contents are pretty irrelevant, right?) I'd just pass `{'name': Section('name')}` here and assert that the file just has `[name]` in it (I think that's the expected result)

makes the test shorter, more concise